docs/guides/modules/storage.rst

```diff
@@ -45,82 +53,125 @@ Change the theme at startup
         ...
 
 
+Create an offline todo app
+++++++++++++++++++++++++++
+
+.. code-block:: python
+
+    from anvil_extras.storage import indexed_db
+    from anvil_extras.uuid import uuid4
+
+    todo_store = indexed_db.create_store('todos')
+    # create_store() is a classmethod that takes a store_name
+    # it will create another store object inside the browsers IndexedDB
+    # or return the store object if it already exists
+    # the todo_store acts as dictionary like object
+
+    class TodoPage(TodoPageTemplate):
+        def __init__(self, **properties):
+            self.init_components(**properties)
+            self.todo_panel.items = list(todo_store.values())
+
+        def save_todo_btn_click(self, **event_args):
+            if not self.todo_input.text:
+                return
+            id = str(uuid4())
+            todo = {"id": id, "todo": self.todo_input.text, "completed": False}
+            todo_store[id] = todo
+            self.todo_panel.items = self.todo_panel.items + [todo]
+            self.todo_input.text = ""
+
+
```

Comment:
@meatballs - can you just check through this code snippet - see if it makes sense - and whether it provides the right api.
I think it probably does.
There's a little bit of tedium in that you have to store the id as the key and in the value. But I think that's ok...

Reply:
LGTM